### DISCUSSION TITLE
Cannot select block lists

### DISCUSSION THREAD

**Discussion by ntnguyen1234** [celzero/rethink-app#1317] [Q&A] [answered]
Hi. I'm having an issue where I can't select any block lists in version `0.5.5c`. When I click on `On-device block list` -> `Configure` -> Select any list, it still shows "0 blocklists in-use" for me.

I'm sorry I didn't realize my video recording is quite fast so might be hard to follow:

<details>
<summary>Video recording</summary>

https://github.com/celzero/rethink-app/assets/23257168/bcda1a1a-4742-4d54-9c43-9a50e44e0a62

</details>

So what should I do now? I tried to clear all data and uninstall and re-install but I still cannot select them. Currently I have downgraded to version `0.5.5a` and the block lists selection works fine.

**Comment by hussainmohd-a** [comment, marked as answer]
@ntnguyen1234, Starting from v055b, the VPN needs to be connected in order to configure the local blocklists. Could you please initiate the VPN and attempt to configure the on-device blocklist to see if it works for you?

**Reply by ntnguyen1234** [reply]
Thanks. I can select the block lists now.

But now there's another issue. If I create a trusted domain rule for an app, that domain is not blocked in all other apps as well. I've seen this behaviour previously in v0.5.5c but after that I can't reproduce any more. Currently I can. I'm not sure what's happening now.
